server/src/test/java/org/elasticsearch/action/ingest/SimulatePipelineRequestParsingTests.java

```diff
@@ -55,6 +56,9 @@
 
     private IngestService ingestService;
 
+    // Ensure one of two tests will use the deprecated explicit type, and the other will not
+    private boolean explicitTypeToggle = randomBoolean();
```

Comment:
We should avoid relying on randomness for coverage too much, can you make both cases tested all the time?

Reply:
Done, this now tests both with and without `_type` for both test cases.